kinds/role/role_kind.cue

```diff
@@ -0,0 +1,36 @@
+package kind
+
+name:        "Role"
+maturity:    "merged"
+description: "Authz Role definition"
+
+lineage: seqs: [
+	{
+		schemas: [
+			//0.0
+			{
+				// Namespace aka tenant/org id.
+				namespace: string
+
+				// Unique role uid.
+				uid: string
+
+				// Role internal name.
+				id: string
```

Comment:
nit: I think it's very confusing calling this an `id`, as we also have a numeric `id` field in the DB schema for roles.